### DISCUSSION TITLE
How to Filter the data of budibase DB based on the query parameter

### DISCUSSION THREAD

**Discussion by kapilgahlot1998** [Budibase/budibase#5111] [Help]
I have created one API which give me some code in the query parameter. I want to search the data into the db based on the code. and want to show all those information on that page. How Can I do that ??

**Comment by shogunpurple** [comment]
@kapilgahlot1998 you will have to give us some more information - I have no idea what you are asking for here.
- Which API? How are you calling it? Through a REST connector?
- What "code" did you get back in the query parameter? Do you mean the actual query parameters?
- it would be very helpful to attach a few screenshots.

Thanks
